app/src/hooks/utils/useLocalStorage.ts

```diff
@@ -0,0 +1,36 @@
+// Serialized implementation of LocalStorage hook from https://github.com/NoahZinsmeister/hypertext/blob/master/context.tsx
```

Comment:
naoh's stuff is great

Reply:
He writes suspiciously concise stuff very helpful

Reply:
hypertext is also beautiful